### PR TITLE
Add local maven publishing for libHttpClient

### PR DESCRIPTION
**Pull request by jplafonta**

### CODE REVIEW

File: Build/libHttpClient.141.Android.Java/build.gradle [hunk 1]
```diff
@@ -19,3 +22,53 @@ allprojects {
...
+        {
+            maven
+            {
+                url ('file:///' + System.getenv("TFS_DropLocation") + '/Maven')
+            }
+        }
+        else {
+            maven {
+                url 'file:///C:/scratch/Maven'
+            }
```
Comment (matt-vanderkolk) [inline Build/libHttpClient.141.Android.Java/build.gradle]: Do you intend to leave this in? Where does the path c:\scratch come from?

File: Build/libHttpClient.141.Android.Java/build.gradle.template [hunk 2]
```diff
@@ -19,3 +22,53 @@ allprojects {
...
+        {
+            maven
+            {
+                url ('file:///' + System.getenv("TFS_DropLocation") + '/Maven')
+            }
+        }
+        else {
+            maven {
+                url 'file:///C:/scratch/Maven'
+            }
```
Comment (matt-vanderkolk) [inline Build/libHttpClient.141.Android.Java/build.gradle.template]: same question

File: Build/libHttpClient.141.Android.Java/build.gradle (hunk 1 above)
Comment (jplafonta) [reply Build/libHttpClient.141.Android.Java/build.gradle]: You are required to set some publish path.  I changed it to set it from the command line, but just chose an arbitrary default location if its not specified

---
In reply to: [223491843](https://github.com/Microsoft/libHttpClient/pull/305#discussion_r223491843) [](ancestors = 223491843)

Comment (lubeltra) [reply Build/libHttpClient.141.Android.Java/build.gradle]: can you make the default path relative to the libHttpClient location?

File: Build/libHttpClient.141.Android.Java/build.gradle.template [hunk 3]
```diff
@@ -19,3 +22,57 @@ allprojects {
...
+                        dependency.appendNode('version', it.version)
+                    }
+                }
+            }
+        } 
+    } 
+
+    repositories { 
+        if (project.hasProperty("PublishRemote") && (System.getenv("VSTS_ENV_ACCESS_TOKEN") != null || project.hasProperty("vstsMavenAccessToken")))
+        {
```
Comment (matt-vanderkolk) [inline Build/libHttpClient.141.Android.Java/build.gradle.template]: >{ [](start = 8, length = 1)

What's with the multiple brace styles?

**Pull request merged**